src/PresentationalComponents/Shield/Shield.js

```diff
@@ -0,0 +1,63 @@
+import { QuestionIcon, SecurityIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import React from 'react';
+import { impactList, colorList } from './consts';
+
+class Shield extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    getColoredBadgeByImpact() {
+        if (impactList.hasOwnProperty(this.props.impact)) {
+            return {
+                icon: <SecurityIcon size={ this.props.size } color={ impactList[this.props.impact].color } />,
```

Comment:
We have to have some kind of `aria-label` that describes what the Badge is representing.

This means title needs to be a required field.

It should display like `aria-label = 'Impact High'`